templates/data/mongodb/what-is-mongodb.html

```diff
@@ -0,0 +1,711 @@
+        </p>
+        <p>MongoDB stores records as documents (specifically BSON documents) gathered together in collections.</p>
+        <p>
+          Scaling the database horizontally can be achieved in MongoDB through the <b>sharding</b> and <b>replica set features</b>. These scalability features also have benefits for fault tolerance in a MongoDB database deployment. The diagram illustrates the sharding model.
+        </p>
+        <div class="p-image-wrapper">
+          <div class="p-image-container u-hide--small" style="padding: 20px;">
+            {{ image(url="https://assets.ubuntu.com/v1/15776dc5-mongo db what is it - diagram_v2.png",
+                        alt="Diagram for MongoDB sharding",
+                        width="3009",
+                        height="883",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-image-container__image"}) | safe
+            }}
+          </div>
+        </div>
+        <p>Sharding is a method for distributing data across multiple machines.</p>
+        <hr class="is-muted" />
+          <ul class="p-list--divided">
+            <li class="p-list__item has-bullet">
+              Client &mdash; an application server that uses the drivers such as python, javascript, C#, C++ and more to integrate with the MongoDB router.
+            </li>
+            <li class="p-list__item has-bullet">
+              Router (mongos) &mdash; a mangos operator that acts as an interface between client applications and the sharded cluster.
+            </li>
+            <li class="p-list__item has-bullet">Enabling searches across numerous attributes</li>
+            <li class="p-list__item has-bullet">
+              Config-server (mongod) &mdash; stores metadata and configuration settings for the cluster.
+            </li>
+            <li class="p-list__item has-bullet">
+              Multiple shards (mongod) &mdash; contains a subset of the sharded data, wherein each shard can be deployed as a replica set. The multiple shards represent a horizontal scale-out architecture since shards are deployed in multiple servers.
+            </li>
+          </ul>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50 p-section--shallow">
+      <hr />
+      <div class="col">
+        <h2>Feature breakdown</h2>
+      </div>
+      <div class="col">
+        <div class="p-image-wrapper u-hide--small">
+          <div class="p-image-container is-highlighted">
+            {{ image(url="https://assets.ubuntu.com/v1/432a1417-Encryption.png",
+                        alt="",
+                        width="350",
+                        height="350",
+                        hi_def=True,
+                        loading="lazy") | safe
+            }}
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row is-split-on-medium">
```

Comment:
It looks like `is-split-on-medium` was deprecated: https://vanillaframework.io/docs/patterns/grid